<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navigation3DControls</class>
 <widget class="QWidget" name="Navigation3DControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>86</width>
    <height>324</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="tbMoveLeft">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/images/move-left_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="tbMoveRight">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/images/move-right_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="tbMoveDown">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/images/move-down_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="tbMoveUp">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/images/move-up_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="tbMoveBack">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/images/move-back_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="tbMoveFront">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/images/move-front_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="tbRotateXNeg">
     <property name="text">
      <string>Rx-</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="tbRotateXPos">
     <property name="text">
      <string>Rx+</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QToolButton" name="tbRotateYNeg">
     <property name="text">
      <string>Ry-</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="tbRotateYPos">
     <property name="text">
      <string>Ry+</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QToolButton" name="tbRotateZNeg">
     <property name="text">
      <string>Rz-</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QToolButton" name="tbRotateZPos">
     <property name="text">
      <string>Rz+</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QToolButton" name="tbScaleXNeg">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/shrink-x_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QToolButton" name="tbScaleXPos">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/grow-x_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QToolButton" name="tbScaleYNeg">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/shrink-y_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QToolButton" name="tbScaleYPos">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/grow-y_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QToolButton" name="tbScaleZNeg">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/shrink-z_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QToolButton" name="tbScaleZPos">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/grow-z_32x32.png</normalon>
      </iconset>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
